File: sql/plsql/Antlr4cs/PlSqlLexerBase.cs
using Antlr4.Runtime;

    public abstract class PlSqlLexerBase : Lexer
    {
        public PlSqlLexerBase self;
        
        public PlSqlLexerBase(ICharStream input)
            : base(input)
        {
            self = this;
        }

        public bool IsNewlineAtPos(int pos)
        {
            int la = _input.La(pos);
            return la == -1 || la == '\n';
        }
    }

File: query/sql/plsql/Dart/PlSqlLexerBase.dart
import 'package:antlr4/antlr4.dart';
import 'dart:io';
import 'dart:convert';

abstract class PlSqlLexerBase extends Lexer
{
    PlSqlLexerBase self;

    PlSqlLexerBase(CharStream input) : super(input)
	{
        self = this;
    }

    bool IsNewlineAtPos(int pos)
    {
        int la = inputStream.LA(pos);
        return la == -1 || la == '\n';
    }
}
